README.md

```diff
@@ -1,144 +1,54 @@
-1. Kończenie gry przez limit czasowy - gracz, któremu skończył się czas przegrywa. Nie rozpatrujemy przypadku remisu.
-1. Dwie wersje językowe gry — Polska i Angielska.
-1. Ładowanie stanu początkowego z np. notacji FEN (https://pl.wikipedia.org/wiki/Notacja_Forsytha-Edwardsa) i/lub tablicy emojis.
-1. Zaimplementuj różne strategie Pat opisane na Wikipedii: https://pl.wikipedia.org/wiki/Pat Np. stare zasady mówiły, że pat powodował wygraną strony, która zapatowała przeciwnika.
-1. Głosowe wykonywanie ruchów. Np. gracz mówi: "Pion na A3".
-
-## Dodatkowe zadania (wykraczające poza zakres kursu):
-
-1. Rozgrywka z komputerem - sztuczna inteligencja. Można wykorzystać np. algorytm minimax. Skorzystajcie z źródeł:
-   - https://alialaa.com/blog/tic-tac-toe-js-minimax
-   - https://www.youtube.com/watch?v=l-hh51ncgDI
-   - https://www.freecodecamp.org/news/simple-chess-ai-step-by-step-1d55a9266977/
-   - https://towardsdatascience.com/create-ai-for-your-own-board-game-from-scratch-minimax-part-2-517e1c1e3362
-   - https://dev.to/zeyu2001/build-a-simple-chess-ai-in-javascript-18eg
-1. Wykonanie testów E2E, przy użyciu odpowiedniego narzędzia. Proponujemy np. Cypress.
-
-Wszelkie inne dodane przez Was funkcjonalności czy usprawnienia infrastrukturalne należy przedstawić w README.md projektu :)
-Template znajdziecie w poprzednim projekcie.
-
-## Porady odnośnie do projektu
-
-- Warto zapozań się z możliwościami kończenia partii szachowych (nie wszystko jest wymagane).
-  - [How Chess Games Can End: 8 Ways Explained](https://www.chess.com/article/view/how-chess-games-can-end-8-ways-explained)
-- Jako inspirację można wykorzystać też działanie szachów zaimplementowanych przez zespół mentora Marcina Wosia na poprzednim CodersCamp (UWAGA: Wymagania mogły ulec zmianie)
-  - https://captainobjective.github.io/chessGame/
-- Dla testów może warto wykorzystać trochę reprezentacji wizualnej. Co powiecie na emoji :) ? Post mentora Mateusza Nowaka.
-  - [Emoji-Driven Development | ZycieNaKodach.pl](https://zycienakodach.pl/tdd-dsl-szachy-emojis)
+![acient chess](https://user-images.githubusercontent.com/57037612/108262231-f110a400-7164-11eb-9c6a-776a54950eb1.jpg)
+
+## Overview
+
+**ACIENT CHESS is another project carried out by our team as part of the CodersCamp.**
+
+## Our Crew
+
+Mentor **[Mateusz Nowak](https://github.com/nowakprojects)**
+
+- [Piotr Rynio](https://github.com/PiotrWR)
+- [Anna Lamperska](https://github.com/lamparina)
+- [Tomasz Dworniczak](https://github.com/tomdworniczak)
+- [Paweł Szambelan](https://github.com/Szambelan)
+- [Dominika Zwolenik](https://github.com/DomiZet)
+
+
+## Functionalities
+
+1. Moving all pieces (making a move, as well as showing possible ones):
+2. Promoting a pawn to another figure (except the king) at the end of the board
+3. Castling
+4. Checking king
+5. Checkmate
+6. Performing the whole chess game
+7. Ending the game by Pat
+
+**Our game  is fully covered by Unit Tests!**
```

Comment:
Zamiast `fully` to dobrze pokazać raport - na górze readme było badge, które prowadzi do raportu. Lepiej go nie usuwać :) To taki standard w repozytoriach jak chce się zaprezentować pokrycie testami.

Reply:
Czy możesz proszę wrzucić znowu jakąś mapkę? Czuję się tu trochę jak w labiryncie. Dodam ten raport razem z linkiem i screenami apki :)